botfront/imports/ui/components/stories/common/BotResponsesContainer.jsx

```diff
@@ -168,6 +187,26 @@ const BotResponsesContainer = (props) => {
                         onToggleButtonType={handleToggleQuickReply}
                         responseType={typeName}
                     />
+                    {initialValue && !initialValue.isNew && getSequence().length === 1 && !checkContentEmpty(getSequence()[0])
+                         && (
+                             <Dropdown
+                                 button
+                                 icon={null}
+                                 compact
+                                 data-cy='import-from-lang'
+                                 className='import-from-lang'
+                                 options={otherLanguages}
+                                 text='Copy from'
```

Comment:
it would also fix the following 
<img width="485" alt="Screen Shot 2020-08-13 at 12 20 08 PM" src="https://user-images.githubusercontent.com/49006070/90160215-799a2c00-dd5f-11ea-8b81-f02dc374cd23.png">

Reply:
I feel like the paste icon isn't explicit enough for what the button is doing, keeping the text is better i think